doc/ikfast_tutorial.rst

```diff
@@ -53,7 +53,7 @@ Thanks to instructions from Stéphane Caron in https://scaron.info/teaching/trou
  make -j4
  sudo make install
 
-Working commit numbers 5cfc7444... comfirmed for Ubuntu 14.04 and 9c79ea26... comfirmed for Ubuntu 16.04, according to Stéphane Caron, for 16.04.
+Working commit numbers 5cfc7444... comfirmed for Ubuntu 14.04 and 9c79ea26... comfirmed for Ubuntu 16.04, according to Stéphane Caron.
```

Comment:
I see, I was wondering where the strange sentence structure came from :)
Please also fix the typo "comfirmed" -> "confirmed"